caravel/bin/caravel

```diff
@@ -125,6 +125,34 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default='data_',
```

Comment:
Yeah but 'data_' is completely arbitrary, if you don't want to read all the tables fine, just don't add a default that would work only for you :)